rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/370_profile.yml

```diff
@@ -139,7 +139,7 @@ fetch nested source:
   - gt: { profile.shards.0.fetch.breakdown.next_reader: 0 }
   - gt: { profile.shards.0.fetch.breakdown.load_stored_fields_count: 0 }
   - gt: { profile.shards.0.fetch.breakdown.load_stored_fields: 0 }
-  - match: { profile.shards.0.fetch.debug.stored_fields: [_id, _ignored, _routing, _source] }
+  - match: { profile.shards.0.fetch.debug.stored_fields: [_id, _routing, _source] }
```

Comment:
Similar to above, I wonder if we need to update the skip above, that's what I would expect.

Reply:
I think I forgot about this commit that already made that change: https://github.com/elastic/elasticsearch/commit/19db4903d10689b736da8a60d96530424c1978d7